assets/js/blocks/product-collection/inspector-controls/taxonomy-controls/taxonomy-item.tsx

```diff
@@ -250,6 +251,10 @@ const TaxonomyItem = ( { taxonomy, termIds, onChange }: TaxonomyItemProps ) => {
 		onChange( newTermIds );
 	};
 
+	const decodeHTMLEntities = ( value: string ) => {
+		return decodeEntities( value ) || '';
```

Comment:
What's the need for the wrapping function and `|| ''` part? Seems like [`decodeEntities`](https://www.npmjs.com/package/@wordpress/html-entities#decodeentities) always returns a string.

Reply:
As you can see in [this code](https://github.com/WordPress/gutenberg/blob/9905301c5f783141177f61e37851eb9e7e96b875/packages/html-entities/src/index.js#L18-L21), it may return non-string value. Therefore, Just to be extra careful, I added `|| ''` part.

Reply:
When I implemented same thing for hand-picked products [here](https://github.com/woocommerce/woocommerce-blocks/blob/fd56c6c94df2ea1f7d2434fca3215590af75cbde/assets/js/blocks/product-collection/inspector-controls/hand-picked-products-control.tsx#L100-L110), it resulted in an error. That’s why I added extra check here.